pkg/cloudprovider/cloud.go

```diff
@@ -37,6 +37,8 @@ type Interface interface {
 	Clusters() (Clusters, bool)
 	// Routes returns a routes interface along with whether the interface is supported.
 	Routes() (Routes, bool)
+	// Volumes returns a volumes interface along with whether the interface is supported.
+	Volumes() (Volumes, bool)
```

Comment:
I guess AWS kind of does, although we only support EBS, in that it also has Elastic File System which is an NFS volume.  Though I think this interface is not exposed, so if we're not sure what it should look like and this "works for now", I'd rather get this in to 1.1 as-is, if it means getting volume auto-creation; we can rework the interface later.